scripts/jenkins/TestQuick.sh

```diff
@@ -9,4 +9,9 @@ hoot --version --debug
 
 export HOOT_TEST_DIFF=--diff
 make -s -f Makefile.old services-test-all
-HootTest --quick
+HootTest --exclude=.*ConflateAverageTest.sh \
+         --exclude=.*ExactMatchInputsTest.sh \
+         --exclude=.*RafahConflateTest.sh \
+         --exclude=.*HaitiDrConflateTest.sh \
+         --slow
+
```

Comment:
Since this script no longer does what it says it does (`TestQuick.sh`), should we rename it to `TestSlow.sh` or `TestOnlyTheTestsThatCurrentlyWorkOnCentOS.sh`?  Or leave it and create a new script and have Jenkins use the new script?  I think that the first option is better, rename it to describe what it actually does.